src/caret_analyze/architecture/architecture.py

```diff
@@ -85,7 +85,7 @@ def get_callback(self, callback_name: str) -> CallbackStructValue:
 
     @property
     def callbacks(self) -> Tuple[CallbackStructValue, ...]:
-        return tuple(_.callbacks for _ in self.callback_groups)
+        return tuple(Util.flatten([_.callbacks for _ in self.callback_groups]))
```

Comment:
nits
`[]` is redundant.

```suggestion
        return tuple(Util.flatten(_.callbacks for _ in self.callback_groups))
```

Reply:
Thank you for the suggestion.
Fixed it on [9db7b86](https://github.com/tier4/CARET_analyze/pull/188/commits/9db7b86888cbbf615f1acc60fbf5a6f1627f1c6d)